app/assets/stylesheets/_related.scss

```diff
@@ -53,13 +49,11 @@
   list-style: none;
   margin: 0;
   padding: 0;
-  overflow: hidden
-}
+  overflow: hidden;
 
-@media (min-width: 641px) {
-  .related ul {
-    font-size: 16px;
-    line-height: 1.25
+  @media (min-width: 641px) {
+    @include core-16;
+    line-height: 1.25;
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3069/files#r344819521)